linux_kernel_modules/mangoh/mangoh_yellow_dev.mdef

```diff
@@ -1,10 +1,10 @@
 cflags:
 {
     -DDEBUG
-    -I${MANGOH_ROOT}/linux_kernel_modules/expander
-    -I${MANGOH_ROOT}/linux_kernel_modules/opt300x
-    -I${MANGOH_ROOT}/linux_kernel_modules/bq27xxx
-    -I${MANGOH_ROOT}/linux_kernel_modules/bq25601
+    -I${CURDIR}/../expander
```

Comment:
I think we should be trying to avoid using relative paths outside of a module's/component's/app's directory, when possible, because doing so breaks relocatability.

Reply:
I'm fine with dropping this commit.